wiki/mnmg/Using_Infiniband_for_MNMG.md

```diff
@@ -0,0 +1,332 @@
+# Using Infiniband for Multi-Node Multi-GPU cuML
+
+These instructions outline how to run multi-node multi-GPU cuML on devices with Infiniband. These instructions assume the necessary Infiniband hardware has already been installed and the relevant software has already been configured to enable communication over the Infiniband devices.
+
+## 1. Install UCX 
+
+### From Conda
+
+Note: this package currently requires CUDA9.2. A CUDA10 package is in the works.
+
+`conda install -c conda-forge -c jakirkham/label/ucx cudatoolkit=9.2 ucx-proc=*=gpu ucx python=3.7`
+
+### From Source
+
+Install autogen if it's not already installed:
+```bash
+sudo apt-get install autogen autoconf libtool
+```
+
+Optionaly install `gdrcopy` for faster GPU-Network card data transfer: 
+
+From the [ucx wiki](https://github.com/openucx/ucx/wiki/NVIDIA-GPU-Support), `gdrcopy` can be installed, and might be necessary, to enable faster GPU-Network card data transfer.
+
+Here are the install instructions, taken from [gdrcopy github](https://github.com/NVIDIA/gdrcopy)
+```bash
+git clone https://github.com/NVIDIA/gdrcopy.git
+cd gdrcopy
+make PREFIX=$CONDA_INSTALL_PREFIX CUDA=/usr/local/cuda
```

Comment:
I added all install at the end of make statement, and isn't this necessary to install this as well? (under the directory specified by PREFIX).

Reply:
Ah. Yes, it definitely should include install. Copied this from the gdrcopy GitHub and not sure how I missed that.